internal/configprovider/manager_test.go

```diff
@@ -197,15 +196,6 @@ map:
   k1: v1
 `},
 				"invalid_yaml_str": {Value: ":"},
-				"valid_yaml_byte_slice": {Value: []byte(`
```

Comment:
Why is this being removed?

Reply:
We dropped the support for the configsources to retrieve bytes, only strings are supported. That is fine since the bytes were converted to strings anyway in the `confmap.Conf` while unmarshaling.